kiliautoml/utils/constants.py

```diff
@@ -1,29 +1,34 @@
-from dataclasses import dataclass
 import os
+from typing_extensions import Literal
 
 
 HOME = os.path.join(os.getenv("HOME"), ".cache", "kili", "automl")  # type: ignore
 
 
-@dataclass
 class ContentInput:
     Checkbox = "checkbox"
     Radio = "radio"
 
 
-@dataclass
+ContentInputT = Literal["checkbox", "radio"]
```

Comment:
a comment: I don't like the repetition, but I do not have any solution (or is it using a `dataclass` 🙃 ?)